packages/keyring-eth-simple/src/simple-keyring.ts

```diff
@@ -27,8 +27,14 @@ type KeyringOpt = {
   version?: SignTypedDataVersion | string;
 };
 
+type Wallet = {
+  privateKey: Buffer;
+  publicKey: Buffer;
+};
```

Comment:
nit suggestion,

```suggestion
type KeyPair = {
  privateKey: Buffer;
  publicKey: Buffer;
};
```

Reply:
Even though the name might be better, I think that we should keep the original name (this PR was meant to only fix eslint warning/errors, and this one is not). Is that ok?

Reply:
sure